src/widgets/JudgePanel.ts

```diff
@@ -309,11 +309,21 @@ export class JudgePanel extends BoxPanel {
       return;
     }
 
+    this.model.submissionStatus = {
+      type: 'progress',
+      runCount: 0,
+      totalCount: 0
+    };
+
```

Comment:
before `getTestCase` request, update submission status in order to show status UI to user